cds/modules/deposit/api.py

```diff
@@ -787,6 +797,33 @@ def generate_duration(self):
         self['duration'] = '{0:02d}:{1:02d}:{2:06.3f}'.format(
             int(hours), int(minutes), seconds)
 
+    def _create_tags(self):
+        """Create additional tags."""
+        objs = [o for o in sorted_files_from_bucket(self._bucket)
+                if o.key.endswith('.vtt')]
```

Comment:
`if o.key.startswith(<MASTER_KEY>) and o.key.endswith('.vtt')` or as regex `/<MASTER_KEY>_[a-zA-Z][a-zA-Z].vtt/`

This is required, to allow other .vtt files that are not subtitles.